### PR TITLE
Fixed prow e2e Test flakes

### PR DESCRIPTION
**Pull request by dtaylor113**
- Updated modal-annotations test to force refresh as opposed to waiting for change

These were failing for me locally, now all e2e passing.

### CODE REVIEW

File: frontend/integration-tests/tests/performance.scenario.ts [hunk 1]
```diff
@@ -32,7 +32,7 @@ const chunkedRoutes = OrderedMap<string, {section: string, name: string}>()
   .set('limit-range', {section: 'Administration', name: 'Limit Ranges'})
   .set('custom-resource-definition', {section: 'Administration', name: 'CRDs'})
   .set('catalog', {section: 'Catalog', name: 'Developer Catalog'})
-  .set('operator-hub', {section: 'Catalog', name: 'OperatorHub'});
+  .set('operator-hub', {section: 'Catalog', name: 'Operator Hub'});
```
Comment (spadgett) [inline frontend/integration-tests/tests/performance.scenario.ts]: This was recently renamed. I think what's in master is correct

**Comment by spadgett** [comment]
@dtaylor113 thanks, I appreicate your looking at these tests

File: frontend/integration-tests/tests/performance.scenario.ts (hunk 1 above)
Comment (dtaylor113) [reply frontend/integration-tests/tests/performance.scenario.ts]: so, my cluster is old?  so I should revert this change?

File: frontend/integration-tests/tests/modal-annotations.scenario.ts [hunk 2]
```diff
@@ -114,6 +114,7 @@ describe('Modal Annotations', () => {
     isPresent: boolean
   ) {
     await crudAnnotationFromDetail(action, annotationKey, annotationValue);
+    await browser.get(`${appHost}/k8s/ns/${testName}/deployments/${WORKLOAD_NAME}`);
```
Comment (spadgett) [inline frontend/integration-tests/tests/modal-annotations.scenario.ts]: It feels like we should at least wait for the loading indicator to go away after changing the URL.

I'm a little unclear why setting the URL is needed. If this helps with test reliability, great. Just want to understand :)

File: frontend/integration-tests/tests/performance.scenario.ts (hunk 1 above)
Comment (spadgett) [reply frontend/integration-tests/tests/performance.scenario.ts]: Yeah, you should take out this change

File: frontend/integration-tests/tests/modal-annotations.scenario.ts (hunk 2 above)
Comment (dtaylor113) [reply frontend/integration-tests/tests/modal-annotations.scenario.ts]: It was taking some undetermined amount of time for the '# Annotations' link to update after creating or deleting an annotation.  Setting the URL instantly updates the correct number in the annotations link.

**Comment by dtaylor113** [comment]
Hi @spadgett, is this ready to go?  -thanks

**Comment by dtaylor113** [comment]
@spadgett, merge pls? -thanks

**Pull request merged**